### ISSUE TITLE
Return the path name in API RTSP session GET request 

### ISSUE THREAD

**Issue by jakew009** [bluenviron/mediamtx#1962]
## Describe the feature

At the moment the API request for `http://localhost:9997/v2/rtspsessions/list` returns something like this:

```
{
  "pageCount": 0,
  "items": [
    {
      "id": "string",
      "created": "string",
      "remoteAddr": "string",
      "state": "idle",
      "bytesReceived": 0,
      "bytesSent": 0
    }
  ]
}
```

Would it be possible to add the name of the path the session is connected to? At the moment you have to make multiple API requests to look it up.

**Comment by aler9** [comment]
Hello, this is fixed by #2022 2022

**Comment by aler9** [comment]
added in v0.23.8
